app/mixins/render-component.js

```diff
@@ -5,7 +5,7 @@ export default Mixin.create({
 	didInsertElement() {
 		if (this._placeholderElement) {
 			this._placeholderElement.parentNode.insertBefore(this.element, this._placeholderElement);
-			$(this._placeholderElement).remove();
+			this._placeholderElement.remove();
```

Comment:
Seems like node.remove() might not be supported by IE11 (https://developer.mozilla.org/pl/docs/Web/API/ChildNode/remove) - how about this._placeholderElement.parentElement.removeChild(this._placeholderElement)?

Reply:
I don't think we support IE11 on mobile